dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelf.scala

```diff
@@ -77,20 +73,14 @@ case class BinaryOutcomeDLCWithSelf(
     outcomeLose: String,
     oraclePubKey: ECPublicKey,
     preCommittedR: ECPublicKey,
-    fundingLocalPrivKey: ECPrivateKey,
-    fundingRemotePrivKey: ECPrivateKey,
-    cetLocalPrivKey: ECPrivateKey,
-    cetRemotePrivKey: ECPrivateKey,
-    finalLocalPrivKey: ECPrivateKey,
-    finalRemotePrivKey: ECPrivateKey,
+    localExtPrivKey: ExtPrivateKey,
+    remoteExtPrivKey: ExtPrivateKey,
```

Comment:
Shouldn't this be a `ExtPubKey` ?

Reply:
No, I haven't separated things out yet, as it currently stands, this class is responsible for building and signing DLC transactions, not for actually representing communication between two parties. This will change of course.

Reply:
This is just replacing many private keys with one